----- Exception Start -----
Exception: Time:2018-03-31 22:41:39  File:/var/Website/Socket/test/forward.php  Line:121  Message:Cannot access protected property Core\Forward::$_pidList
#0 unknow {closure}(unknow)
#1 /var/Website/Socket/core/Forward.php call_user_func(434)
#2 /var/Website/Socket/core/Forward.php Core\Forward->_msgHandleFromParent(624)
#3 unknow Core\Forward->monitorParent(unknow)
#4 /var/Website/Socket/event/Select.php call_user_func_array(255)
#5 /var/Website/Socket/event/Select.php Event\Select::_loopForIo(167)
#6 /var/Website/Socket/core/Forward.php Event\Select::loop(606)
#7 /var/Website/Socket/core/Forward.php Core\Forward->_forkForChild(647)
#8 /var/Website/Socket/core/Forward.php Core\Forward->fork(1107)
#9 /var/Website/Socket/test/forward.php Core\Forward->run(129)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-01 13:36:28  File:/var/Website/Socket/connection/TraitConnection.php  Line:29  Message:不是 socket 客户端链接
#0 /var/Website/Socket/core/Forward.php Connection\TcpConnection->__construct(913)
#1 unknow Core\Forward->Core\{closure}(unknow)
#2 /var/Website/Socket/event/Select.php call_user_func_array(255)
#3 /var/Website/Socket/event/Select.php Event\Select::_loopForIo(167)
#4 /var/Website/Socket/core/Forward.php Event\Select::loop(1120)
#5 /var/Website/Socket/core/Forward.php Core\Forward->loop(1161)
#6 /var/Website/Socket/test/forward.php Core\Forward->run(125)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 09:33:40  File:/var/Website/Socket/core/Forward.php  Line:406  Message:Argument 1 passed to Core\Forward::_msgHandleFromParent() must be of the type array, null given, called in /var/Website/Socket/core/Forward.php on line 591
#0 /var/Website/Socket/core/Forward.php Core\Forward->_msgHandleFromParent(591)
#1 unknow Core\Forward->monitorParent(unknow)
#2 /var/Website/Socket/event/Select.php call_user_func_array(255)
#3 /var/Website/Socket/event/Select.php Event\Select::_loopForIo(167)
#4 /var/Website/Socket/core/Forward.php Event\Select::loop(744)
#5 /var/Website/Socket/core/Forward.php Core\Forward->fork(1227)
#6 /var/Website/Socket/test/forward.php Core\Forward->run(128)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 09:33:40  File:/var/Website/Socket/core/Forward.php  Line:298  Message:Argument 1 passed to Core\Forward::_msgHandleFromChild() must be of the type array, null given, called in /var/Website/Socket/core/Forward.php on line 573
#0 /var/Website/Socket/core/Forward.php Core\Forward->_msgHandleFromChild(573)
#1 unknow Core\Forward->monitorChild(unknow)
#2 /var/Website/Socket/event/Select.php call_user_func_array(255)
#3 /var/Website/Socket/event/Select.php Event\Select::_loopForIo(167)
#4 /var/Website/Socket/core/Forward.php Event\Select::loop(744)
#5 /var/Website/Socket/core/Forward.php Core\Forward->fork(1227)
#6 /var/Website/Socket/test/forward.php Core\Forward->run(128)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 09:33:40  File:/var/Website/Socket/core/Forward.php  Line:298  Message:Argument 1 passed to Core\Forward::_msgHandleFromChild() must be of the type array, null given, called in /var/Website/Socket/core/Forward.php on line 573
#0 /var/Website/Socket/core/Forward.php Core\Forward->_msgHandleFromChild(573)
#1 unknow Core\Forward->monitorChild(unknow)
#2 /var/Website/Socket/event/Select.php call_user_func_array(255)
#3 /var/Website/Socket/event/Select.php Event\Select::_loopForIo(167)
#4 /var/Website/Socket/core/Forward.php Event\Select::loop(1206)
#5 /var/Website/Socket/core/Forward.php Core\Forward->loop(1247)
#6 /var/Website/Socket/test/forward.php Core\Forward->run(128)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 09:33:40  File:/var/Website/Socket/core/Forward.php  Line:406  Message:Argument 1 passed to Core\Forward::_msgHandleFromParent() must be of the type array, null given, called in /var/Website/Socket/core/Forward.php on line 591
#0 /var/Website/Socket/core/Forward.php Core\Forward->_msgHandleFromParent(591)
#1 unknow Core\Forward->monitorParent(unknow)
#2 /var/Website/Socket/event/Select.php call_user_func_array(255)
#3 /var/Website/Socket/event/Select.php Event\Select::_loopForIo(167)
#4 /var/Website/Socket/core/Forward.php Event\Select::loop(744)
#5 /var/Website/Socket/core/Forward.php Core\Forward->fork(1227)
#6 /var/Website/Socket/test/forward.php Core\Forward->run(128)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 09:33:41  File:/var/Website/Socket/core/Forward.php  Line:406  Message:Argument 1 passed to Core\Forward::_msgHandleFromParent() must be of the type array, null given, called in /var/Website/Socket/core/Forward.php on line 591
#0 /var/Website/Socket/core/Forward.php Core\Forward->_msgHandleFromParent(591)
#1 unknow Core\Forward->monitorParent(unknow)
#2 /var/Website/Socket/event/Select.php call_user_func_array(255)
#3 /var/Website/Socket/event/Select.php Event\Select::_loopForIo(167)
#4 /var/Website/Socket/core/Forward.php Event\Select::loop(744)
#5 /var/Website/Socket/core/Forward.php Core\Forward->fork(1227)
#6 /var/Website/Socket/test/forward.php Core\Forward->run(128)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 17:06:26  File:/var/Website/Socket/core/Forward.php  Line:523  Message:Argument 1 passed to Core\Forward::monitorChild() must be an instance of Event\EvCtrl\EventCtrl, instance of Event\EventCtrl\SelectCtrl given
#0 unknow Core\Forward->monitorChild(unknow)
#1 /var/Website/Socket/event/Select.php call_user_func_array(270)
#2 /var/Website/Socket/event/Select.php Event\Select::_loopForIo(182)
#3 /var/Website/Socket/core/Forward.php Event\Select::loop(1134)
#4 /var/Website/Socket/core/Forward.php Core\Forward->loop(1178)
#5 /var/Website/Socket/test/forward.php Core\Forward->run(120)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 21:48:06  File:/var/Website/Socket/core/Forward.php  Line:523  Message:Argument 1 passed to Core\Forward::monitorChild() must be an instance of Event\EvCtrl\EventCtrl, instance of Event\EventCtrl\SelectCtrl given
#0 unknow Core\Forward->monitorChild(unknow)
#1 /var/Website/Socket/event/Select.php call_user_func_array(278)
#2 /var/Website/Socket/event/Select.php Event\Select::_loopForIo(190)
#3 /var/Website/Socket/core/Forward.php Event\Select::loop(1140)
#4 /var/Website/Socket/core/Forward.php Core\Forward->loop(1184)
#5 /var/Website/Socket/test/forward.php Core\Forward->run(120)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 22:38:16  File:/var/Website/Socket/core/Forward.php  Line:638  Message:Call to a member function send() on resource
#0 /var/Website/Socket/core/Forward.php Core\Forward->_forkForChild(684)
#1 /var/Website/Socket/core/Forward.php Core\Forward->fork(1197)
#2 /var/Website/Socket/test/forward.php Core\Forward->run(120)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 22:38:16  File:/var/Website/Socket/core/Forward.php  Line:638  Message:Call to a member function send() on resource
#0 /var/Website/Socket/core/Forward.php Core\Forward->_forkForChild(684)
#1 /var/Website/Socket/core/Forward.php Core\Forward->fork(1197)
#2 /var/Website/Socket/test/forward.php Core\Forward->run(120)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 22:38:16  File:/var/Website/Socket/core/Forward.php  Line:638  Message:Call to a member function send() on resource
#0 /var/Website/Socket/core/Forward.php Core\Forward->_forkForChild(684)
#1 /var/Website/Socket/core/Forward.php Core\Forward->fork(1197)
#2 /var/Website/Socket/test/forward.php Core\Forward->run(120)
----- Exception End ------

----- Exception Start -----
Exception: Time:2018-04-02 22:38:16  File:/var/Website/Socket/core/Forward.php  Line:638  Message:Call to a member function send() on resource
#0 /var/Website/Socket/core/Forward.php Core\Forward->_forkForChild(684)
#1 /var/Website/Socket/core/Forward.php Core\Forward->fork(1197)
#2 /var/Website/Socket/test/forward.php Core\Forward->run(120)
----- Exception End ------

